### DISCUSSION TITLE
Newsletter sobre carreira em desenvolvimento de software

### DISCUSSION THREAD

**Discussion by woliveiras** [frontendbr/forum#940] [Dicas]
Galera, estou criando uma newsletter sobre carreira em desenvolvimento de software onde vou compartilhar dicas, novidades e sugestões sobre carreira na área, seja sobre programação, design, gestão, etc.

Para se inscrever é só entrar [neste link](http://bit.ly/newsletter-dev-career)

Se tiverem dicas, sobre como melhor atender as pessoas que estiverem se inscrevendo na lista, podem me enviar aqui também. :heart:

**Comment by felipefialho** [comment]
Muito bom @woliveiras o/
